src/devices/xiaomi.ts

```diff
@@ -2558,7 +2637,55 @@ const definitions: Definition[] = [
         exposes: [e.switch().withEndpoint('l1'), e.switch().withEndpoint('l2'),
             e.power(), e.current(), e.energy(), e.voltage(), e.device_temperature(),
         ],
-        extend: [xiaomi.extend.switchType],
+        extend: [
+            xiaomi.extend.switchType(),
+            xiaomi.extend.powerOnBehavior({
+                lookup: {'on': 0, 'previous': 1, 'off': 2, 'toggle': 3},
+            }),
+            xiaomi.extend.operationMode({
+                description: 'Decoupled mode for 1st relay',
+                endpoint: 'l1',
+            }),
+            xiaomi.extend.operationMode({
+                description: 'Decoupled mode for 2nd relay',
+                endpoint: 'l2',
+            }),
+            extend.binary({
+                name: 'interlock',
+                valueOn: true,
+                valueOff: false,
+                cluster: 'aqaraOpple',
+                attribute: {id: 0x02d0, type: 0x10},
+                description: 'Enabling prevents both relays being on at the same time (Interlock)',
+                zigbeeCommandOptions: {manufacturerCode},
+            }),
+            extend.enumLookup({
+                name: 'mode',
+                lookup: {'power': 0, 'pulse': 1, 'dry': 3},
+                cluster: 'aqaraOpple',
+                attribute: {id: 0x0289, type: 0x20},
+                description: 'Work mode: Power mode, Dry mode with impulse, Dry mode',
+                zigbeeCommandOptions: {manufacturerCode},
+            }),
+            extend.numeric({
+                name: 'pulse_length',
+                valueMin: 200,
+                valueMax: 2000,
+                unit: 'ms',
+                cluster: 'aqaraOpple',
+                attribute: {id: 0x00eb, type: 0x21},
+                description: 'Impulse length in Dry mode with impulse',
+                zigbeeCommandOptions: {manufacturerCode},
+            }),
+            xiaomi.extend.action({
```

Comment:
Instead of 2x action here, can you try:

```js
extend.actionEnumLookup({lookup: {'single': 1}, cluster: 'genMultistateInput',attribute: 'presentValue'})
```

        lookup: {'single': 1},
        cluster: 'genMultistateInput',
        attribute: 'presentValue',
        description: 'Input',